src/v/pandaproxy/schema_registry/seq_writer.cc

```diff
@@ -228,6 +228,73 @@ ss::future<bool> seq_writer::write_config(
     });
 }
 
+ss::future<std::optional<bool>> seq_writer::do_delete_config(
+  subject sub, model::offset write_at, seq_writer& seq) {
+    vlog(plog.debug, "delete config sub={} offset={}", sub, write_at);
+
+    try {
+        co_await seq._store.get_compatibility(sub, default_to_global::no);
+    } catch (const exception&) {
+        // subject config already blank
+        co_return false;
+    }
+
+    std::vector<seq_marker> sequences{
+      co_await _store.get_subject_config_written_at(sub)};
+
+    storage::record_batch_builder rb{
+      model::record_batch_type::raft_data, model::offset{0}};
+
+    std::vector<config_key> keys;
+    for (auto s : sequences) {
```

Comment:
`const auto& s` ?

Reply:
Generally yeah. I see these passed around by value (and in a loop like this) throughout surrounding, so maybe there's a reason for that. Did not bother interrogating here, but could do.

Reply:
> I see these passed around by value (and in a loop like this) throughout surrounding code, so maybe there's a reason for that.

They're not massive types, but I see no reason not to use a `const&` here.